include/odp/api/spec/time.h

```diff
@@ -21,20 +21,24 @@ extern "C" {
 #include <odp/api/time_types.h>
 
 /** @defgroup odp_time ODP TIME
- *  Chip and CPU level wall clock time.
+ *  SoC global and CPU local wall clock time
+ *
  *  @{
  */
 
 /**
  * Current local time
  *
- * Returns current local time stamp value. The local time source provides high
- * resolution time, it is initialized to zero during ODP startup and will not
- * wrap around in at least 10 years.
- * Local time stamps are local to the calling thread and must not be shared
- * with other threads.
+ * Returns current CPU local time stamp value. The value can be used only by the calling thread,
+ * and only as long as it runs on the same CPU. Local time stamp values must not be shared with
+ * other threads.
```

Comment:
Maybe this should just say that the time source of the local time stap value is specific to the calling thread and the current CPU and that time stamps of different time sources cannot be compared or otherwise mixed. But obeying that, the time stamp can be used in any thread (e.g. a thread can store and process time stamps collected by another thread).